src/device/mod.rs

```diff
@@ -1500,6 +1500,9 @@ impl Runtime {
                     ip_addresses: vec![
                         IpAddr::V4(Ipv4Addr::new(10, 5, 0, 1)),
                         IpAddr::V4(Ipv4Addr::new(100, 64, 0, 1)),
+                        IpAddr::V6(Ipv6Addr::new(
```

Comment:
:thinking: I am not sure, so this field is generally representing the IP addresses of the peer about which the event is. And this `match` case is specifically for VPN node, which has static IP addresses. But the server itself does _not_ have IPv6 address set. Therefore we should probably not report it maybe? Even though the IP is reserved for VPN.

Reply:
Deleted with [fix](https://github.com/NordSecurity/libtelio/pull/96#discussion_r1334050218)